database/migrations/2024_12_10_014452_create_participant_healthy_marriage_introductory_survey.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('participant_healthy_marriage_introductory_survey', function (Blueprint $table) {
+            $table->id();
+            $table->foreignId('participant_id')->constrained();
+            $table->date('date_of_birth')->nullable();
+            $table->string('fatherhood_program');
+            $table->boolean('become_responsible_father')->default(false);
+            $table->boolean('referred')->default(false);
+            $table->boolean('court_ordered')->default(false);
+            $table->boolean('address_child_support_concerns')->default(false);
+            $table->string('other_reason_for_attendance')->nullable();
+            $table->boolean('word_of_mouth')->default(false);
```

Comment:
Let's adapt this to referred_by_word_of_mouth, referred_by_past_participant, etc.